library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/recorder/Producer.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.recorder
+
+internal interface Producer
```

Comment:
why empty interface is needed now? Is it just a placeholder for now?

Reply:
Just a placeholder yes.

Reply:
Actually this is not needed anymore.